API.md

```diff
@@ -16,38 +17,45 @@ An object for encapsulating an Amplify Geo transform request and Amplify credent
 
 ### Parameters
 
-*   `sourceName` **[String][7]** A user defined name used for determining the maplibre data source and the maplibre layers
-*   `data` **([Array][12]\<Coordinate> | [Array][12]\<Feature>)** An array of coordinate data or GeoJSON Features used as the data source for maplibre
-*   `map` **maplibreMap** A maplibre map on which the points will be drawn
-*   `options` **[Object][13]** An object containing options for changing the styles and features of the points rendered to the map, see the options for more details on available settings
+*   `sourceName` **[String][8]** A user defined name used for determining the maplibre data source and the maplibre layers
+*   `data` **([Array][13]\<Coordinate> | [Array][13]\<Feature>)** An array of coordinate data or GeoJSON Features used as the data source for maplibre
+*   `map` **maplibre-gl-js-Map** A maplibre-gl-js [map][10] on which the points will be drawn
+*   `options` **[Object][14]** An object containing options for changing the styles and features of the points rendered to the map, see the options for more details on available settings
 
-    *   `options.showCluster` **[String][7]** Determines whether or not points close together should be clustered into a single point (optional, default `true`)
-    *   `options.clusterOptions` **[String][7]** Object for determining cluster options, see [ClusterOptions][14] for more details (optional, default `{}`)
-    *   `options.unclusteredOptions` **[String][7]** Object for determining cluster options, see [UnclusteredOptions][15] for more details (optional, default `{}`)
-*   `mapStyle` **[String][7]** A string containing the map style data used to draw this map
+    *   `options.showCluster` **[String][8]** Determines whether or not points close together should be clustered into a single point (optional, default `true`)
+    *   `options.clusterOptions` **[String][8]** Object for determining cluster options, see [ClusterOptions][15] for more details (optional, default `{}`)
+    *   `options.unclusteredOptions` **[String][8]** Object for determining unclustered point options, see [UnclusteredOptions][16] for more details (optional, default `{}`)
+*   `mapStyle` **[String][8]** The map style returned from Amplify Geo that is currently being used. This is used to determine the default fonts used by maplibre-gl-js.
 
-Returns **DrawPointsOutput** An object the string id's of the sources and layers used to draw the points to the map
+### Properties
+
+*   `sourceId` **[String][8]** 
+*   `clusterLayerId` **[String][8]** 
+*   `clusterSymbolLayerId` **[String][8]** 
+*   `unclusteredLayerId` **[String][8]** 
+
+Returns **DrawPointsOutput** output An object containing the string id's of the sources and layers used to draw the points to the map. This includes the sourceId, clusterLayerId, clusterSymbolLayerId, unclusteredLayerId.
```

Comment:
Should the properties be after the `DrawPointsOutput` definition? And possible a definition/comment for each of these Ids.

Reply:
Because of the way the documentation js works I can't get those properties to come after the `Return` section